web/settings.py

```
@@ -359,6 +390,56 @@
+# AI Settings
+AI_PROVIDER = 'gemini'  # or 'demo'
+AI_TEMPERATURE = 0.7
+AI_MAX_TOKENS = 1024
+GEMINI_MODEL = 'gemini-2.0-flash'  # Using the new flash model
+
+# OpenAI Settings
+OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY', '')
+OPENAI_MODEL = os.environ.get('OPENAI_MODEL', 'gpt-3.5-turbo')
+
+# Log AI configuration status
+import logging
```

Ruff (0.8.2)

Module level import not at top of file

(E402)